spec/requests/unwithdraw_spec.rb

```diff
@@ -1,52 +1,71 @@
 # frozen_string_literal: true
 
 RSpec.describe "Unwithdraw" do
-  let(:edition) { create(:edition, :withdrawn, :political, :past_government) }
+  let(:managing_editor) { create(:user, managing_editor: true) }
+  let(:withdrawn_edition) { create(:edition, :withdrawn) }
+  let(:history_mode_edition) { create(:edition, :withdrawn, :political, :past_government) }
```

Comment:
I don't think this variable fully explains what it's actually creating:

```suggestion
  let(:withdrawn_edition_in_history_mode) { create(:edition, :withdrawn, :political, :past_government) }
```

Reply:
sure